en/docs/administer/configure-developer-portal-self-sign-up.md

```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
  
 Users can click **LOGIN/SIGN UP** and then click **Create an account** to sign up to your Developer Portal without having to send you a request to register them.
 
-![Create an account](../assets/img/administration/self-sign-up/create-an-account.png){.cInlineImage-half}        
+![Create an account](../assets/img/administration/self-sign-up/create-an-account.png){.cInlineImage-half} 
+
+## Approve user accounts
+
+To check the users who try to access your Developer Portal via self-sign-up, you can follow either of the following approaches:
+
+- Enable auto-approval so that every user who signs up is allowed to access your Developer Portal by default.
+- Manually check each sign-up and approval or reject it as required.
+
+### Enable auto-approval
+
+To enable auto-approval, follow the steps given below:
+
+1. Click **Settings** in the left navigation menu of the Choreo Console.
+
+2. Click **Self Signups**. 
+
+3. Toggle the **Auto-approval** switch.  This enables your account to automatically approve user accounts that create accounts to access your Developer Portal.
```

Comment:
If we have this info in the introductory sentence, lets remove "This enables your account to automatically approve user accounts that create accounts to access your Developer Portal."